src/assets/javascripts/application.js

```diff
@@ -302,9 +302,10 @@ function initialize(config) { // eslint-disable-line func-style
 
         /* Retrieve all results */
         const links = Array.prototype.slice.call(
-          document.querySelectorAll("[data-md-component=search] [href]"))
-        if (!links.length)
-          return
+          document.querySelectorAll(
+            ".md-search__input, [data-md-component=search] [href]"
```

Comment:
The removal: makes tab behaviour consistent before you have made any search.
e.g. on master, load your site, hit tab several times. Then load your site again, make a search (so results show up), then hit tab several times. There is a behaviour difference and the removal was part of making the behaviour consistent.

The addition: lets you tab back to the search input box (otherwise how does a keyboard user go back to that box to refine their search?)